gensim/test/test_hdpmodel.py

```diff
@@ -47,7 +47,6 @@ def testfile():
     return os.path.join(tempfile.gettempdir(), 'gensim_models.tst')
 
 
-
 class TestHdpModel(unittest.TestCase):
```

Comment:
should it inherit from TestBaseTopicModel too? Feel free to add the missing methods